Modules/Forum/classes/class.ilObjForumListGUI.php

```diff
@@ -138,18 +141,26 @@ public function getProperties()
             $lpCont = "<a class=\"il_ItemProperty\" target=\"" . ilFrameTargetInfo::_getFrame('MainContent') .
                 "\" href=\"ilias.php?baseClass=ilRepositoryGUI&amp;cmd=viewThread&amp;cmdClass=ilobjforumgui&amp;target=true&amp;pos_pk=" .
                 $last_post['pos_pk'] . "&amp;thr_pk=" . $last_post['pos_thr_fk'] . "&amp;ref_id=" .
-                $this->ref_id . "#" . $last_post["pos_pk"] . "\">" .
+               (int) $this->ref_id . "#" . $last_post["pos_pk"] . "\">" .
                 ilObjForumAccess::prepareMessageForLists($last_post['pos_message']) . "</a> " .
                 strtolower($this->lng->txt('from')) . "&nbsp;";
 
+            global $DIC;
```

Comment:
Please try to move the `global $DIC` to the constructor and assign it to a class property. Then, access the class property here.